cmd/fyne/internal/commands/package.go

```diff
@@ -360,11 +360,10 @@ func (p *Packager) validate() (err error) {
 		return errors.New("parameter -sourceDir is currently not supported for mobile builds. " +
 			"Change directory to the main package and try again")
 	}
-	os.Chdir(p.srcDir)
 
 	p.appData.CustomMetadata = p.customMetadata.m
 
-	data, err := metadata.LoadStandard(p.srcDir)
+	data, err := metadata.LoadStandard(baseDir)
```

Comment:
I disagree with this change. The .toml file should be read from the source dir over the current directory.
If we go with basedir instead then builds can change based on current directory and the app metadata is detached from the app it describes.

Reply:
In my opinion it is better to run fyne commands where FyneApp.toml is. Most of the build systems needs configuration or script in root folder. If you are concerning about multiple main programs in one project here is my suggestion. fyne will first look at the srcdir for FyneApp.toml as you said but if it couldn't find it there, it will look for it in the cwd where fyne command runs. I still respect your decision.

Reply:
Also I only have access to windows so I can't make required changes for linux and especially macos. It is better to close this pr and make yourself when you have time.